Src/IronPython/Modules/sys.cs

```diff
@@ -751,6 +751,14 @@ public class floatinfo : PythonTuple {
             dict["stdin"] = dict["__stdin__"];
             dict["stdout"] = dict["__stdout__"];
             dict["stderr"] = dict["__stderr__"];
+            
+            if(RuntimeInformation.IsOSPlatform(OSPlatform.Windows)) {
+                dict["platform"] = "win32";
+            } else if(RuntimeInformation.IsOSPlatform(OSPlatform.Linux)) {
+                dict["platform"] = "posix";
+            } else {
+                dict["platform"] = "darwin";
```

Comment:
From a future compatibility point of view, not sure if darwin should be the fallback. I know it's just the 3 platforms right now, but in the future who knows what might get added to .NET Core (or if we support .NET Standard there might be even more platforms). Anyway, I'd make it an explicit check for MacOS and maybe fallback to win32, or unknown, or maybe even cli?